File: back-end/BusinessLogic/Models/TaskModel.cs
namespace BusinessLogic.Models
{
	public class TaskModel
	{
		public int Id { get; set; }
		public string Title { get; set; }
		public string Description { get; set; }
		public string Priority { get; set; }
		public string State { get; set; }
		public string TimeToFinish { get; set; }
		public string CreationDate { get; set; }
		public IEnumerable<EmployeeModel> Employees { get; set; }
	}
}

File: client/src/app/tasks-grid/task-card/task.model.ts
export interface Task {
  id: number;
  title: string;
  description: string;
  priority: string;
  state: string;
  timeToFinish: string;
  creationDate: string;
  employees: Employee[];
}

export interface Employee {
  id: number;
  name: string;
  surname: string;
  workLoad: string;
  taskIds: number[];
}
